terraform/modules/app/variables.tf

```diff
@@ -150,3 +150,17 @@ variable "task_role_arn" {
 variable "execution_role_arn" {
   type = string
 }
+
+variable "environment" {
+  type = string
+}
+
+variable "workspace" {
```

Comment:
I'm guessing this is var because we're still doing some odd special-case stuff with the default workspace (calling it `ecs`?) If so then then's fine for now - otherwise would be good to just use `terraform.workspace`.

Reply:
Yes indeed - we use that as ECS and some for Redis we use shared. so i am sticking to what we have and then call it using local.workspace